client/rpc/src/state/state_light.rs

```diff
@@ -199,6 +199,14 @@ impl<Block, F, B, E, RA> StateBackend<B, E, Block, RA> for LightState<Block, F,
 		Box::new(result(Err(client_err(ClientError::NotAvailableOnLightClient))))
 	}
 
+	fn storage_next_key(
+		&self,
+		_block: Option<Block::Hash>,
+		_key: StorageKey,
+	) -> FutureResult<Option<StorageKey>> {
+		Box::new(result(Err(client_err(ClientError::NotAvailableOnLightClient))))
```

Comment:
Not sure if it is possible to implement this on light client

Reply:
@svyatonik ?

Reply:
`NotAvailableOnLightClient` should be enough for now.